### DISCUSSION TITLE
Choria builds for Ubuntu Noble Numbat (24.04 LTS)

### DISCUSSION THREAD

**Discussion by timdeluxe** [choria-io/general#50] [General]
Looks like there are no choria packages for Ubuntu Noble Numbat (24.04 LTS) yet. Are there plans already on when the could be available?

**Comment by ripienaar** [comment]
We follow Puppet so I dont think they support it yet.  Choria being built for AIO Puppet, little point in us supporting what they dont.

https://www.puppet.com/docs/pe/2023.7/supported_operating_systems.html

**Reply by nur-dmelchers** [reply]
In the open source version, the puppet agent is available: https://www.puppet.com/docs/puppet/8/system_requirements.html#supported_operating_systems
but the server is not: https://www.puppet.com/docs/puppet/8/server/install_from_packages.html#supported-operating-systems

**Reply by ripienaar** [reply]
Ah I missed it there thanks, it's a bummer as I just did the release builds (not announced yet) yesterday

Generally community contributes this

We need correct lines in:

https://github.com/choria-io/go-choria/blob/main/.github/workflows/release.yaml
https://github.com/choria-io/go-choria/blob/main/packager/buildspec.yaml
https://github.com/choria-io/packager ( I will do this today )

Can you test if the previous ubuntu debs work when installed on the new one? Then its really easy else we need to maybe do some distro specific stuff

**Reply by Tuxdiver** [reply]
I can confirm, that
`ii  choria         0.29.3+jammy amd64        The Choria Orchestrator Server and Broker`
works on ubuntu 24.04
"mco ping" shows the ubuntu 24.04 server.

**Reply by ripienaar** [reply]
OK, I've done the packager stuff, if you want/can please do a PR for those other 2 parts I mentioned.  If you look in the history of the files you'll see what got added for recent Debian for example

**Reply by timdeluxe** [reply]
@ripienaar 
> ... if you want/can please do a PR for those other 2 parts I mentioned ...

Who is "you"? Shall i? It really looks pretty simple.

**Reply by ripienaar** [reply]
Yeah, @timdeluxe or @Tuxdiver whoever feel they can pick it up :)

**Reply by ripienaar** [reply]
I made a issues for this https://github.com/choria-io/general/issues/51, please tag the PR with that like others in history

**Reply by timdeluxe** [reply]
Here you go: https://github.com/choria-io/go-choria/pull/2185 - but i am confused about tagging, do you mean a git tag? A label? A prefix?

**Reply by ripienaar** [reply]
just commit messages like `(#51) Support ubuntu blah blah`

**Reply by timdeluxe** [reply]
Yeah, but #51 is in "general", while my commit is in "go-choria". I am not a github specialist, but i guess the mapping wont work correctly, does it?

**Reply by ripienaar** [reply]
ugh, yeah hold on sorry messed up making the issue, will transfer it.

**Reply by ripienaar** [reply]
https://github.com/choria-io/go-choria/issues/2186 so `(#2186) ....`

**Reply by timdeluxe** [reply]
Alright. Done.
